taas/taas.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+"""An implementation of Tests as a Service (TAAS) application."""
+
+import connexion
+from behave.__main__ import main as behave_main
+
+from os import listdir
+from os.path import isfile, join
+
+TEST_DIR = "features"
+TEST_SUFFIX = ".feature"
+
+
+def get_file_list(directory, suffix):
+    """Get list of files from selected directory that has the specified suffix."""
+    # read list of files
+    files = [f for f in listdir(directory) if isfile(join(directory, f))]
+    # filter it
+    files = [f for f in files if f.endswith(suffix)]
+    # remove suffix
+    files = [f[:-len(suffix)] for f in files]
+    # and finally return sorted version of the list
+    return sorted(files)
+
+
+def get_readiness():
+    """Get TaaS service readiness."""
+    return {}, 200
+
+
+def get_liveness():
+    """Get TaaS service liveness."""
+    return {}, 200
```

Comment:
Since both readiness and liveness are the same, do we need both? Maybe single `health_check` function would do the trick here?